doc/changes/latest.inc

```diff
@@ -157,6 +159,8 @@ Enhancements
 
 Bugs
 ~~~~
+- Fix bug with :func:`mne.io.read_raw_edf` where new symbols for interpreting data in microvolt was added (:gh:`9187` **by new contributor** |Sumalyo Datta|_ )
```

Comment:
```suggestion
- Fix bug with :func:`mne.io.read_raw_edf` where µV was not correctly recognized (:gh:`9187` **by new contributor** |Sumalyo Datta|_)
```

Reply:
Thanks for the suggestion, I have committed the changes